ZCRMiOS/Classes/ZCRMSubformRecord.swift

```diff
@@ -13,7 +13,7 @@ public class ZCRMSubformRecord : ZCRMEntity
 	public var modifiedTime : String = APIConstants.STRING_MOCK
 	public var createdTime : String = APIConstants.STRING_MOCK
     var layout : ZCRMLayout?
-	public var fieldNameVsValue : [String:Any?] = [String:Any?]()
+	public var fieldNameVsValue : [String:Any] = [String:Any]()
```

Comment:
Any? vs Any while getting values will this work fine?

Reply:
Yes. Any can have optional values.